core_backend/app/llm_call/check_output.py

```diff
@@ -105,17 +104,10 @@ async def _check_align_score(
     }
 
     if align_score.score < float(ALIGN_SCORE_THRESHOLD):
-        error_response = UserQueryResponseError(
-            query_id=llm_response.query_id,
-            error_message=STANDARD_FAILURE_MESSAGE,
-            error_type=ErrorType.ALIGNMENT_TOO_LOW,
-            debug_info=llm_response.debug_info.copy(),
-        )
-        error_response.debug_info["factual_consistency"] = factual_consistency.copy()
-        return error_response
-    else:
-        llm_response.debug_info["factual_consistency"] = factual_consistency.copy()
-        return llm_response
+        llm_response.llm_response = ""
```

Comment:
Here we can replace the LLM response with `STANDARD_FAILURE_MESSAGE` right? Better than blank. I would also rewrite that response to say:

    STANDARD_FAILURE_MESSAGE = (
        "Sorry, I am unable to find an answer to your question in my knowledge base."

    )

And remove the "Please rephrase your question or ask a different one." - what to do next should be left up to the chat manager.

Reply:
@suzinyou what do you think of this? This makes the most sense from a chat manager perspective (like our flow in Botpress)

Reply:
Is the idea to send only reason for error, and chat manager handles error message to send to user? If so leaving llm_response blank seems to make sense?

Reply:
wait, why don't we return an ErrorResponse instance if the check fails?? If we send error response and error code about Alignscore check failure, chat managers and handle the user-facing message?

Reply:
Because we want the user to still be able to browse the FAQs even if the summarization didn't work well. For that, we need to send the `content_response` field, which we only send when we respond without erroring out. 

I think it makes sense for the alignscore to just be an internal cleaner of the LLM response. On botpress, I can also check if the LLM response is the standard "didn't find anything" response and replace that message with whatever I want while still having access to the FAQs.

Reply:
The only problem is that we create coupling between botpress and AAQ. So every time we change the STANDARD_FAILURE_MESSAGE, we'll need to update botpress. 

Keeping it blank means, Botpress can see that no LLM response was provided and do as it sees fit. It may be even better to return None instead of "".

What are your thoughts @amiraliemami and @suzinyou?

Reply:
And agree with @amiraliemami that sending back an ErrorResponse means it's an error. It shouldn't be an error unless we want it handled like one. Should just be like the other rails and modify the response.

Reply:
I see, makes sense re. the coupling. "None" would work!

Reply:
Ok I'll make it None.